docs/htdocs/info/docs/tools/vep/script/vep_example.html

```diff
@@ -372,6 +368,297 @@ <h4> Changing memory requirements</h4>
 
 
   
+  <h2 id="pangenomes">Pangenome assemblies</h2>
+
+  <p> Generically speaking, VEP is able to analyse variants in <b>any species or assembly</b> (even if not part of <a href="/info/about/species.html">Ensembl data</a>) by providing your own <a href="vep_cache.html#fasta">FASTA file</a> and <a href="vep_cache.html#gff" target="_blank">GFF/GTF annotation</a>: </p>
```

Comment:
<p>VEP is able to analyse variants in <b>any species or assembly</b> (even if not part of <a href="/info/about/species.html">Ensembl data</a>) by providing your own <a href="vep_cache.html#fasta">FASTA file</a> and <a href="vep_cache.html#gff" target="_blank">GFF/GTF annotation</a>: </p>